content/03.results.md

```diff
@@ -134,27 +134,27 @@ Following SNV consensus calling (**Figure {@fig:S1}** and **Figure {@fig:S2}A-G*
-We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs, displayed as other embryonal tumors) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].
+We detected canonical _SMARCB1/SMARCA4_ deletions or inactivating mutations in atypical teratoid rhabdoid tumors (ATRTs) and C19MC amplification in the embryonal tumors with multilayer rosettes (ETMRs, displayed as "Other embryonal tumors" in **Figure {@fig:Fig2}B**) [@doi:10.1007/s00401-020-02182-2; @doi:10.1093/neuonc/noab178; @doi:10.1186/s40478-020-00984-9; @doi:10.1038/nature22973].
 
 #### High-grade gliomas
 <!--SAMPLECOUNT-->
-Across HGGs, we found that _TP53_ (57%, 35/61) and _H3F3A_ (52%, 32/61) were both most mutated and co-occurring genes (**Figure {@fig:Fig2}A and C**), followed by frequent mutations in _ATRX_ (30%, 18/61), a gene commonly mutated in gliomas [@doi:10.1080/14728222.2018.1487953].
+Across HGGs, we found that _TP53_ (57%, 36/63) and _H3F3A_ (54%, 34/63) were both most mutated and co-occurring genes (**Figure {@fig:Fig2}A and C**), followed by frequent mutations in _ATRX_ (30%, 18/61), a gene commonly mutated in gliomas [@doi:10.1080/14728222.2018.1487953].
 We observed recurrent amplifications and fusions in _EGFR_, _MET_, _PDGFRA_, and _KIT_, highlighting that these tumors utilize multiple oncogenic mechanisms to activate tyrosine kinases, as has been previously reported [@doi:10.1002/ijc.32258; @doi:10.1016/j.ccell.2017.08.017; @doi:10.1186/s40478-020-00905-w].
 Gene set enrichment analysis showed upregulation (ANOVA Bonferroni-corrected p < 0.01) of DNA repair, G2M checkpoint, and MYC pathways as well as downregulation of the TP53 pathway (**Figure {@fig:Fig5}B**).
 The two tumors with ultra-high TMB (> 100 Mutations/Mb) were from patients with known mismatch repair deficiency syndrome [@doi:10.1093/neuonc/noz192].
 
 #### Other CNS tumors
 <!--SAMPLECOUNT-->
 We observed that 25% (15/60) of ependymoma tumors were _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion-positive ependymomas [@:doi.org/10.1038/nature13109] and that 68% (21/31) of craniopharyngiomas were driven by mutations in _CTNNB1_ (**Figure {@fig:Fig2}D**).
-Multiple histologies contained somatic mutations or fusions in _NF2_, including 41% (7/17) of meningiomas, 5% (3/60) of ependymomas, and 27% (3/11) schwannomas.
-We observed rare fusions in _ERBB4_, _YAP1_, _KRAS_, and _MAML2_ in 10% (6/60) of ependymoma tumors.
+Multiple histologies contained somatic mutations or fusions in _NF2_, including 41% (7/17) of meningiomas, 5% (3/60) of ependymomas, and 25% (3/12) of schwannomas.
+We observed rare fusions in _ERBB4_, _YAP1_, and _MAML2_ in 10% (6/60) of ependymoma tumors.
```

Comment:
```suggestion
We observed rare fusions in _ERBB4_, _YAP1_, and/or _QKI_ in 10% (6/60) of ependymoma tumors.
```

I am going to update this to these genes since YAP1 fuses with MAML2 and MAMLD1, to not double count.